modules/ROOT/pages/create-community.adoc

```diff
@@ -81,7 +81,7 @@ Chatter tracking enables your community to create discussions about specific obj
 
 . In *Setup*, search for *Chatter* using the *Quick Find* box and click *Feed Tracking*.
 . Click *CommunityAPI*.
-. Without selecting any fields, click *Enable feed tracking*. 
+. Without selecting any fields, click *Enable feed tracking*.
```

Comment:
Does one "select" a field? Typically, fields are populated rather than chosen....

Reply:
Out of scope for this PR, but in scope for the rewrite of this page in https://www.mulesoft.org/jira/browse/DOCS-11514 .